code/modules/client/preferences_mysql.dm

```diff
@@ -449,4 +452,14 @@
 		load_character(C)
 		return 0
 	load_character(C,pick(saves))
-	return 1*/
\ No newline at end of file
+	return 1*/
+
+/datum/preferences/proc/SetChangelog(client/C,hash)
+	lastchangelog=hash
+	var/DBQuery/query = dbcon.NewQuery("UPDATE client SET lastchangelog='[lastchangelog]' WHERE ckey='[C.ckey]'")
```

Comment:
this won't work, replace it with `UPDATE [format_table_name("player")] SET lastchangelog='[lastchangelog]' WHERE ckey='[C.ckey]'`